dex/networks/eth/swap/contract.go

```diff
@@ -1,7 +1,7 @@
 // Code generated - DO NOT EDIT.
 // This file is a generated binding and any manual changes will be lost.
 
-package eth
+package swap
```

Comment:
No objection to the move.  In fact this was recently discussed and it will eventually be needed to have these files versioned to avoid collisions.

https://github.com/decred/dcrdex/pull/1301#discussion_r757660939

Although it'll actually need to be versioned **folders** at that point to have versioned packages.

Reply:
You mean v0 v1 v2 folders?